src/next/middleware/http-auth.ts

```diff
@@ -0,0 +1,35 @@
+import {NextRequest} from "next/server";
+
+/**
+ * Handles basic authentication
+ */
+export function handleAuth (
```

Comment:
`handleHttpBasicAuth` or `handleAuth`? The changelog is referencing `handleHttpBasicAuth`.

Reply:
Ah yeah, meant to rename it